src/chainparams.cpp

```diff
@@ -476,6 +500,12 @@ class CCustomParams : public CRegTestParams {
     {
         strNetworkID = chain;
         UpdateFromArgs(args);
+
+        if (!anyonecanspend_aremine) {
+            assert("Anyonecanspendismine was marked as false, but they are in the genesis block"
+                    && initialFreeCoins == 0);
```

Comment:
```
chainparams.cpp:506:24: error: ‘initialFreeCoins’ was not declared in this scope
```

Reply:
Yeah, it's a WIP PR, so there's plenty of loose ends still.